### PR TITLE
BXMSDOC-3436: Managing & monitoring process server for DM/PAM 7.2

### PR DESCRIPTION
**Pull request by gadeyredhat**
[Jira Id](https://issues.jboss.org/browse/BXMSDOC-3436)

[PAM Rendered Output - Process Server](http://file.pnq.redhat.com/~gadey/BXMSDOC-3436-process-server-pam/)
[DM Rendered Output - Process Server](http://file.pnq.redhat.com/~gadey/BXMSDOC-3436-process-server-dm/)
[PAM Rendered Output - Interacting with processes and tasks](http://file.pnq.redhat.com/~gadey/BXMSDOC-3436-pam/)

### CODE REVIEW

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Tasks* or  click *Menu* -> *Track* -> *Task Inbox*.
+. On the *Manage Tasks* page or the *Task Inbox* page, click the *Advanced Filters* icon.
+. In the *Advanced Filters* pane, enter the name and description of the filter, and click *Add New*.
+. Select the `name` attribute from the *Select column* drop-down list. The content of the drop-down will change to *name != value1*.
+. Click the drop-down again and choose *equals to* as the logical query.
+. Change the value of the text field to the name of the task.
+. Click *Save* and the filter is applied on the current task list.
+. Click *Show/hide columns* on the upper right of the tasks list and the task variables of the specified process id will be displayed.
```
Comment (MarianMacik) [inline doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: `task variables of the specified process id` - it should be `of the specified task`

**Review by MarianMacik — COMMENTED** [review]
I think that Activation/Deactivation of KIE containers, Interacting with Processes and Tasks (whole section named assembly_interacting-with-processes) shouldn't be available in RH DM documentation. @mswiderski is that right? I am mainly asking about KIE containers activation/deactivation (primarily used for process instances).

Smart Router TLS section is fine for both RHDM/RHPAM.

**Comment by mswiderski** [comment]
@MarianMacik yes, activation and deactivation currently applies to PAM only

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc (hunk 1 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: Changed as suggested.

**Comment by gadeyredhat** [comment]
@mswiderski, @MarianMacik, I have applied conditional text so that the "KIE containers activation/deactivation" is available for PAM documentation only.

**Comment by MarianMacik** [comment]
@gadeyredhat I also think that Interacting with Processes and Tasks (whole section named assembly_interacting-with-processes) shouldn't be available in RH DM documentation.

**Comment by gadeyredhat** [comment]
Hi @MarianMacik, Interacting with Processes and Tasks is a PAM only document and is not available in DM documentation. Sorry, I missed this in my earlier reply.

**Comment by MarianMacik** [comment]
@gadeyredhat I was just checking since there is also a DM Rendered Output for that section.

**Review by MarianMacik — APPROVED** [review]

File: doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+.Prerequisite
+A container has been created and configured in {CENTRAL}.
+
+.Procedure
+. Log in to {CENTRAL}.
+. In the main menu, click *Menu* -> *Deploy* -> *Execution Servers*.
+. Select your server from the *Server Configurations* section on the left side of the page.
+. Select the deployment unit (KIE container) you want to activate/deactivate under the *Deployment Units* section on the left.
+. Click *Activate* or *Deactivate* at the upper right hand corner.
```
Comment (csherrar) [inline doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc]: IBMSG 369 - Replace "at" with "in", "upper right hand" with "upper-right", and add additional text information when providing a location.

s/Click *Activate* or *Deactivate* at the upper right hand corner./Click *Activate* or *Deactivate* in the upper-right corner of the deployment unit pane.

File: doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+You can now stop the creation of new process instances from a given container by deactivating it but at the same time continue working on its existing process instances and tasks. In case the deactivation is temporary, you can activate the container again later. The activation or deactivation of KIE containers do not require restarting of KIE server.
+
+.Prerequisite
+A container has been created and configured in {CENTRAL}.
+
+.Procedure
+. Log in to {CENTRAL}.
+. In the main menu, click *Menu* -> *Deploy* -> *Execution Servers*.
+. Select your server from the *Server Configurations* section on the left side of the page.
```
Comment (csherrar) [inline doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc]: IBMSG - 211, 369

s/Select your server from the *Server Configurations* section on the left side of the page./From the Server Configurations pane, which is on the left of the page, select your server.

File: doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+. Select the deployment unit (KIE container) you want to activate/deactivate under the *Deployment Units* section on the left.
```
Comment (csherrar) [inline doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc]: IBMSG - 209, 211
s/Select the deployment unit (KIE container) you want to activate/deactivate under the *Deployment Units* section on the left./From the Deployment Units pane, select the deployment unit you want to activate or deactivate.

File: doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tsl-support-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
+[id='kie-server-smart-router-enable-tsl-support-proc']
+= Configuring Smart Router for TSL support
+
+You can now configure Smart Router (previously, KIE Server Router) for TSL support to enable HTTPS traffic through it.
```
Comment (csherrar) [inline doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tsl-support-proc.adoc]: Just a suggestion, do you think it reads better?

s/You can now configure Smart Router (previously, KIE Server Router) for TSL support to enable HTTPS traffic through it./You can now configure Smart Router (previously, KIE Server Router) for TSL support to allow HTTPS traffic.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,17 @@
+[id='interacting-with-processes-process-instances-special-filter-proc']
+= Defining special filters for process instances
+
+You can create special process instance filters using the *Advanced* filters option in {CENTRAL}. The process instance list view changes when a filter over the column `processId` is defined. Once you create and apply the special filter, you can view the process instance variables from the specific process id as columns using the *Show/hide columns* option.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: I shortened this a little bit, and I think it is clearer for me, what do you think?

You can use the *Advanced* filters option in {CENTRAL} to create special filters for process instances. When you create a special filter with 'processId' defined, you can view the process instance variables of the process as columns using *Show/hide columns*.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the *Advanced Filters* icon.
+. In the *Advanced Filters* pane, enter the name and description of the filter, and click *Add New*.
+. Select the `processId` attribute from the *Select column* drop-down list. The content of the drop-down will change to *processId != value1*.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: IBMSG - 209
s/Select the `processId` attribute from the *Select column* drop-down list. The content of the drop-down will change to *processId != value1*./From the *Select column* list, select the 'processId' attribute. The value will change to *processId != value1*.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+. Click the drop-down again and choose *equals to* as the logical query.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: s/Click the drop-down again and choose *equals to* as the logical query./From the *Select column* list, select *equals to* for the logical query.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+. Change the value of the text field to the name of the process id.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: s/Change the value of the text field to the name of the process id./In the text field, enter the name of the process id.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc [hunk 10]
```diff
@@ -0,0 +1,17 @@
+[id='interacting-with-processes-tasks-special-filter-proc']
+= Defining special filters for tasks
+
+You can create special filters for tasks in *Manage Tasks* and *Task Inbox* using the *Advanced* filters option. You can create a special filter that will provide domain specific columns to be added to a task list and the task variables are enabled as columns. Once you create and apply the special filter, you can view the task variables of the task as columns using the *Show/hide columns* option.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Tasks* or  click *Menu* -> *Track* -> *Task Inbox*.
+. On the *Manage Tasks* page or the *Task Inbox* page, click the *Advanced Filters* icon.
+. In the *Advanced Filters* pane, enter the name and description of the filter, and click *Add New*.
+. Select the `name` attribute from the *Select column* drop-down list. The content of the drop-down will change to *name != value1*.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: IBMSG - 209
s/Select the `name` attribute from the *Select column* drop-down list. The content of the drop-down will change to *name != value1*./From the *Select column* list, select the 'name' attribute. The value will change to *name != value1*.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+. Click the drop-down again and choose *equals to* as the logical query.
+. Change the value of the text field to the name of the task.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: s/Change the value of the text field to the name of the task./In the text field, enter the name of the task.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+= Defining special filters for tasks
+
+You can create special filters for tasks in *Manage Tasks* and *Task Inbox* using the *Advanced* filters option. You can create a special filter that will provide domain specific columns to be added to a task list and the task variables are enabled as columns. Once you create and apply the special filter, you can view the task variables of the task as columns using the *Show/hide columns* option.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Manage* -> *Tasks* or  click *Menu* -> *Track* -> *Task Inbox*.
+. On the *Manage Tasks* page or the *Task Inbox* page, click the *Advanced Filters* icon.
+. In the *Advanced Filters* pane, enter the name and description of the filter, and click *Add New*.
+. Select the `name` attribute from the *Select column* drop-down list. The content of the drop-down will change to *name != value1*.
+. Click the drop-down again and choose *equals to* as the logical query.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: s/Click the drop-down again and choose *equals to* as the logical query./From the *Select column* list, select *equals to* for the logical query.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc [hunk 13]
```diff
@@ -0,0 +1,17 @@
+[id='interacting-with-processes-tasks-special-filter-proc']
+= Defining special filters for tasks
+
+You can create special filters for tasks in *Manage Tasks* and *Task Inbox* using the *Advanced* filters option. You can create a special filter that will provide domain specific columns to be added to a task list and the task variables are enabled as columns. Once you create and apply the special filter, you can view the task variables of the task as columns using the *Show/hide columns* option.
```
Comment (csherrar) [inline doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: I shortened this a little bit, and I think it is clearer for me, what do you think?

You can use the Advanced filters option in {CENTRAL} to create special filters for tasks in *Manage Tasks* and *Task Inbox*. When you create a special filter with the task defined, you can view the task variables of the task as columns using *Show/hide columns*.

**Review by csherrar — COMMENTED** [review]
Some suggestions but please let me know what you think. I added the page numbers for IBM Style Guide related suggestions.

File: doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc (hunk 3 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc (hunk 4 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc (hunk 2 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/kie-server-deactivate-kie-containers-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tsl-support-proc.adoc (hunk 5 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/admin-and-config/kie-server-smart-router-enable-tsl-support-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc (hunk 6 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: Changed as suggested. I agree that shortening this is a good idea and it sure is clearer.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc (hunk 7 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc (hunk 8 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc (hunk 9 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-process-instances-special-filter-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc (hunk 13 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: The shortened para indeed reads better. Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc (hunk 10 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc (hunk 12 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: Changed as suggested.

File: doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc (hunk 11 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/processes/interacting-with-processes-tasks-special-filter-proc.adoc]: Changed as suggested.

**Comment by gadeyredhat** [comment]
All SME, Peer & QE comments implemented.

**Pull request merged**